### ISSUE TITLE
hack/update-codegen.sh is missing #!/bin/bash shebang line

### ISSUE THREAD

**Issue by marcindulak** [awslabs/aws-service-operator#195]
https://github.com/awslabs/aws-service-operator/blob/76312848693937324f5920d771cde1abe2f51fdd/hack/update-codegen.sh#L1

```
make rebuild
aws-service-operator-codegen process
INFO[0000] Model Path: models/                          
INFO[0000] cloudformationtemplate.yaml                  
INFO[0000] dynamodb.yaml                                
INFO[0000] ecrrepository.yaml                           
INFO[0000] elasticache.yaml                             
INFO[0000] s3bucket.yaml                                
INFO[0000] snssubscription.yaml                         
INFO[0000] snstopic.yaml                                
./hack/update-codegen.sh
./hack/update-codegen.sh: 3: set: Illegal option -o pipefail
Makefile:36: recipe for target 'k8s-codegen' failed
make: *** [k8s-codegen] Error 2
```

See e.g. https://askubuntu.com/questions/886537/set-e-o-pipefail-not-working-on-bash-script-on-ubuntu-16

**Comment by mhausenblas** [comment]
Thanks for reporting this @marcindulak and please stand by for some next steps, here.

**Comment by computeracer** [comment]
@mhausenblas is there any updates on this? I found adding the shebang to the script helped when self compiling as  well.

**Comment by marcindulak** [comment]
I think the solution of calling bash explicitly in the Makefile, as in the PR is better, since Mac OS users typically won't use the default path to bash https://itnext.io/upgrading-bash-on-macos-7138bd1066ba

**Comment by mhausenblas** [comment]
Yeah, we can use the PR, however, please note that we're in the process of preparing the Phase II of ASO with some re-platforming and hence I'm hesitant to introduce too big changes.

**Comment by computeracer** [comment]
@mhausenblas  Glad to hear of future work on the ASO. Is there anything we can follow on the progress?

**Comment by mhausenblas** [comment]
@computeracer I can, at the current point in time, not (yet) provide a time line but I can tell you that I am working on it, high priority. I will create an issue here and once that is done you and anyone else who's interested in contributing are more than welcome to. The Phase II is really all about community and getting ASO to the next level.
